.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - Help Wanted
+  - Proposal
+  - Pinned
+  - Security
+  - Request for Comment
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
`wontfix` seems to mean "We do not ever plan to fix this for any reason", perhaps `stale` is more appropriate? Since it's possible there was an issue we wanted to address but were not satisfied with the solutions.